packages/filebrowser/src/listing.ts

```diff
@@ -1663,7 +1663,7 @@ export namespace DirListing {
      */
     populateHeaderNode(node: HTMLElement): void {
       let name = this._createHeaderItemNode('Name');
-      let modified = this._createHeaderItemNode('Last Modified');
+      let modified = this._createHeaderItemNode('Hello From Texas');
```

Comment:
Hey, thank you for this pull-request ! You did  a great job. 

an example of feedback could be:

"I believe the F of From should be lower case, woudl you mind updating your pull-request."